recipe/meta.yaml

```diff
@@ -24,9 +24,11 @@ source:
   sha256: 4804320e1e6cbe3a5421997b52199e3c1a3829b2ecb6489641da4b8e32faf500
 
 build:
-  number: 2
+  number: 3
   script_env:
     - OS    # [win]
+  run_exports:
+    - {{ pin_subpackage('tbb', max_pin='2020') }}
```

Comment:
```suggestion
    - {{ pin_subpackage('tbb', upper_bound='2021.0.a0') }}
```

Reply:
@isuruf, thank you, I've applied the suggestion (but removed the dot before `a0`)